### ISSUE TITLE
Accessibility: Unable to trigger "shift+alt+f1" for Open Accessibility Help from terminal

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#186435]
Testing #186217

Platform: Linux Mint
Reader: Orca

**Comment by meganrogge** [comment]
it should just be `alt+f1` - are you seeing something different?

**Comment by meganrogge** [comment]
does `alt+f1` work?

**Comment by karthiknadig** [comment]
Alt+F1 on linux mint is bound to workspace switching. So both do "Alt+f1" or "shift+Alt+f1" don't work.

**Comment by meganrogge** [comment]
i'm pretty surprised this doesn't work as it's what we've used historically - 

```
linux: {
					primary: KeyMod.Alt | KeyMod.Shift | KeyCode.F1,
					secondary: [KeyMod.Alt | KeyCode.F1]
				}
```

@jvesouza, I know you often use Linux. have you encountered this problem as well?

**Comment by jvesouza** [comment]
> 
> @jvesouza, I know you often use Linux. have you encountered this problem as well?

No, it works perfect in my environment. I currently use fedora 38 with gnome.

**Comment by karthiknadig** [comment]
@meganrogge I will test this again today. I couldn't test this before because both commands has same shortcut. I will try with "shift+alt+f1".

**Comment by meganrogge** [comment]
thanks so much @karthiknadig

**Comment by karthiknadig** [comment]
This works now.
